conf/log4j2.properties

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+## Log4j2 file that configures logging for processes other than Accumulo services
+status = info
+dest = err
+name = AccumuloTestingDefaultLoggingProperties
+
+appender.console.type = Console
+appender.console.name = STDERR
+appender.console.target = SYSTEM_ERR
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = %d{ISO8601} [%c{3}] %-5p: %m%n
+
+logger.accumulotesting.name = org.apache.accumulo.testing
+logger.accumulotesting.level = DEBUG
+
+logger.accumulo.name = org.apache.accumulo
+logger.accumulo.level = WARN
+
+logger.hadooptest.name = org.apache.hadoop
+logger.hadooptest.level = WARN
+logger.hadoopmapred.name = org.apache.hadoop.mapreduce
+logger.hadoopmapred.level = ERROR
+logger.hadooputil.name = org.apache.hadoop.util
+logger.hadooputil.level = ERROR
+
+logger.zookeepertest.name = org.apache.zookeeper
+logger.zookeepertest.level = WARN
```

Comment:
The old file had zookeeper as `ERROR` instead of `WARN`. Not sure how big of a deal that is though.